docs/docs/jsx-in-depth.md

```diff
@@ -0,0 +1,312 @@
+---
+id: jsx-in-depth
+title: JSX In Depth
+permalink: docs/jsx-in-depth.html
+---
+
+Fundamentally, JSX just provides syntactic sugar for the `React.createElement(component, props, children)` function. The JSX code:
+
+```js
+<MyComponent prop1={'one'} prop2={'two'}>stuff</MyComponent>
```

Comment:
It's a little weird to me that this uses {} since people don't ever do that in practice.